x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/entity_analytics_management_page_privileges.cy.ts

```diff
@@ -49,6 +51,8 @@ describe(
       );
       cy.get(RISK_SCORE_PRIVILEGES_CALLOUT).should('contain', 'manage_index_templates');
       cy.get(RISK_SCORE_PRIVILEGES_CALLOUT).should('contain', 'manage_transform');
+      cy.get(RISK_SCORE_PREVIEW_PRIVILEGES_CALLOUT);
```

Comment:
I was going to split out a separate test for these assertions but they take ~20-30 seconds to run so I thought better to keep it as one